Packs/GitHub/Integrations/GitHub/GitHub.py

```diff
@@ -1224,6 +1224,69 @@ def get_github_actions_usage():
     return_outputs(readable_output=human_readable, outputs=ec, raw_response=usage_result)
 
 
+def list_check_runs(owner_name, repository_name, run_id, commit_id):
+    url_suffix = None
+
+    if run_id:
+        url_suffix = f"/repos/{owner_name}/{repository_name}/check-runs/{run_id}"
+    elif commit_id:
+        url_suffix = f"/repos/{owner_name}/{repository_name}/commits/{commit_id}/check-runs"
+    else:
+        return_error("You have to specify either the check run id of the head commit refernce")
+
+    check_runs = http_request(method="GET", url_suffix=url_suffix)
+
+    return [r for r in check_runs.get('check_runs')]
+
+
+def get_github_get_check_run():
+    """ List github check runs.
+
+    """
+    command_args = demisto.args()
+    owner_name = command_args.get('owner', '')
+    repository_name = command_args.get('repository', '')
+    run_id = command_args.get('run_id', '')
+    commit_id = command_args.get('commit_id', '')
+
+    check_run_result = []
+
+    check_runs = list_check_runs(owner_name=owner_name, repository_name=repository_name, run_id=run_id, commit_id=commit_id)
+
+    for check_run in check_runs:
+            check_run_id = check_run.get('id', '')
+            check_external_id = check_run.get('external_id', '')
+            check_run_name = check_run.get('name', '')
+            check_run_app_name = check_run['app'].get('name', '')
+            check_run_pr = check_run['pull_requests']
```

Comment:
Are `app` and `pull_requests` always expected to be in `check_run`? Might be better to use `.get(app, {})` and `.get('pull_requests')` instead

Reply:
@DeanArbel app and pull_requests are always there, but will use .get() anyways